### DISCUSSION TITLE
Datalogger Function

### DISCUSSION THREAD

**Discussion by turrengoechea** [TUDelft-CNS-ATM/bluesky#480] [Q&A]
I am working on a function or a command for Bluesky in Python, that can create a file that records the data of the different aircraft's data of the scenario. Is there a function already created within Bluesky Python that I can use? And how can I use it? If is not, how could I create one?

Thank you

**Comment by turrengoechea** [comment]
> I am working on a function or a command for Bluesky in Python, that can create a file that records the data of the different aircraft's data of the scenario. Is there a function already created within Bluesky Python that I can use? And how can I use it? If is not, how could I create one?
> 
> Thank you

**Comment by ProfHoekstra** [comment]
Yes, there is, see this page on the wiki:

<https://github.com/TUDelft-CNS-ATM/bluesky/wiki/Data-Logging>
[bluesky.png]
Data Logging<https://github.com/TUDelft-CNS-ATM/bluesky/wiki/Data-Logging>
github.com<https://github.com/TUDelft-CNS-ATM/bluesky/wiki/Data-Logging>




It might suit your needs.

Best regards/Met vriendelijke groeten,
    Jacco Hoekstra


On 3 Oct 2023, at 10:10, Tomás De Urrengeochea Cantavenera ***@***.***> wrote:

﻿

I am working on a function or a command for Bluesky in Python, that can create a file that records the data of the different aircraft's data of the scenario. Is there a function already created within Bluesky Python that I can use? And how can I use it? If is not, how could I create one?

Thank you
